### PR TITLE
About Us Page

### PR DESCRIPTION
**Pull request by johnperry99**
- Created frontend for About Us page
- Added ButtonLink for AboutUs component to AppHeader.jsx
- Added route for AboutUs component in App.js

### CODE REVIEW

**Review by johnsoncv — APPROVED** [review]
lgtm

File: client/src/AboutUs.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const AboutUs = () => {
+
+    return (
+        <Box sx={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
+            <Card sx={{ padding: 2, marginBottom: 1, marginTop: 1, width: 95 + "%" }}>
+                <Typography variant="h4">About</Typography>
+                <Typography paragraph variant="body1">INFOX comes out of a research project at
+                    Carnegie Mellon University, in collaboration with researchers from the IT University
+                    Copenhagen, Peking University, and the University of Passau, exploring how to improve
+                    ollaboration and coordination in open-source and variant-rich software development.
```
Comment (johnsoncv) [inline client/src/AboutUs.jsx]: typo  
ollaboration -> collaboration

**Review by johnsoncv — CHANGES_REQUESTED** [review]

File: client/src/AboutUs.jsx (hunk 1 above)
Comment (LaithHanania) [reply client/src/AboutUs.jsx]: good catch

File: client/src/AboutUs.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Box, Typography, Card } from "@mui/material";
+import Link from '@mui/material/Link';
+import Button from '@mui/material/Button';
+
+const AboutUs = () => {
+
+    return (
+        <Box sx={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
+            <Card sx={{ padding: 2, marginBottom: 1, marginTop: 1, width: 95 + "%" }}>
```
Comment (LaithHanania) [inline client/src/AboutUs.jsx:10]: for width I think you can do "95%" instead of 95 + %

File: client/src/AboutUs.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    INFOX set out to understand activities in forks and provide insights to
+                    interested observers of a project and all its forks. The current beta release
+                    focuses on simple analytics, but we plan many more functions as we move along.
+                </Typography>
+                <Typography paragraph variant="body1">
+                    INFOX is open source (MIT license) and <Link href="https://github.com/FancyCoder0/INFOX">hosted on GitHub.</Link> Forks and pull requests are
+                    welcome of course. :)
+                </Typography>
+            </Card>
+            <Card sx={{ padding: 2, marginBottom: 1, width: 95 + "%" }}>
```
Comment (LaithHanania) [inline client/src/AboutUs.jsx:31]: width: "95%"

File: client/src/AboutUs.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                <Typography variant="h4">People</Typography>
+                <Typography variant="body1"><Link href="http://luyaoren.com/">Luyao Ren</Link></Typography>
+                <Typography variant="body1"><Link href="https://www.cs.cmu.edu/~shuruiz/">Shurui Zhou</Link></Typography>
+                <Typography variant="body1"><Link href="https://www.cs.cmu.edu/~ckaestne/">Christian Kästner</Link></Typography>
+                <Typography variant="body1"><Link href="http://www.itu.dk/people/wasowski/">Andrzej Wąsowski</Link></Typography>
+            </Card>
+            <Card sx={{ padding: 2, width: 95 + "%" }}>
```
Comment (LaithHanania) [inline client/src/AboutUs.jsx:38]: width: "95%"

File: client/src/AboutUs.jsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                    , <Link href="https://www.infosun.fim.uni-passau.de/spl/people-lessenich.php">O. Leßenich</Link>
+                    , <Link href="http://sei.pku.edu.cn/~xiongyf04/">Y. Xiong</Link>,
+                    <Link href="http://www.itu.dk/people/wasowski/">A. Wąsowski</Link>
+                    , and <Link href="https://www.cs.cmu.edu/~ckaestne/"> C. Kästner</Link>
+                    . <b>Identifying Features in Forks.</b>
+                    In Proceedings of the 40th International Conference on Software Engineering (ICSE),
+                    New York, NY: ACM Press, May 2018. Acceptance rate: 21 % (105/502). [ pdf ]
+                </Typography>
+            </Card>
+            <Card sx={{display:"flex", marginTop:1, width:50+"%", justifyContent:"space-evenly"}}>
```
Comment (LaithHanania) [inline client/src/AboutUs.jsx:65]: width: "50%"

**Review by LaithHanania — CHANGES_REQUESTED** [review]

**Review by LaithHanania — APPROVED** [review]
Looks good!

File: client/src/AboutUs.jsx (hunk 1 above)
Comment (johnperry99) [reply client/src/AboutUs.jsx]: Resolved

**Review by johnsoncv — APPROVED** [review]

**Pull request merged**